test/Cpp/src/multiport/CallbackMultiport.lf

```diff
@@ -0,0 +1,61 @@
+// Test bank of reactors to multiport input with id parameter in the bank.
```

Comment:
The name of the test seems outdated. Maybe PresentPorts.lf? Also the description isn't very clear